pkgs/games/gzdoom/default.nix

```diff
@@ -55,6 +55,15 @@ let
 
     NIX_CFLAGS_LINK = "-lopenal -lfluidsynth";
 
+    desktopItems = [
+      (makeDesktopItem {
+        name = "gzdoom";
+        exec = "gzdoom";
+        desktopName = "GZDoom";
+        categories = "Game;";
+      })
+    ];
+
     installPhase = ''
       install -Dm755 gzdoom "$out/lib/gzdoom/gzdoom"
```

Comment:
```suggestion
      runHook preInstall

      install -Dm755 gzdoom "$out/lib/gzdoom/gzdoom"
```

Reply:
Ah, I made this change locally, but forgot to commit

Reply:
I just fixed it